qa/cephfs/overrides/ignorelist_health.yaml

```diff
@@ -8,6 +8,7 @@ overrides:
       - MDS_DAMAGE
       - MDS_DEGRADED
       - MDS_FAILED
+      - MDS_INSUFFICIENT_STANDBY
```

Comment:
This has started to show up for some reason (debugging now) in pacific too - https://pulpito.ceph.com/lflores-2024-02-15_17:32:20-rados-wip-yuri10-testing-2024-02-08-0854-pacific-distro-default-smithi/7561657/

There other host of warnings are PG_DEGRADED which are new too in fs:upgrade too.

Reply:
Volume got created at:

```
2024-02-15T18:22:52.001 DEBUG:teuthology.orchestra.run.smithi067:> sudo /home/ubuntu/cephtest/cephadm --image docker.io/ceph/ceph:v16.2.4 shell -c /etc/ceph/ceph.conf -k /etc/ceph/ceph.client.admin.keyring --fsid 343b5320-cc2e-11ee-95ba-87774f69a715 -- bash -c 'ceph fs volume create cephfs --placement=4'
```

Warning report check from mon.0:

```
2024-02-15T18:22:55.693+0000 7f2c37c18700  7 mon.smithi067@0(leader).mds e1 print_map
e2
enable_multiple, ever_enabled_multiple: 1,1
compat: compat={},rocompat={},incompat={1=base v0.20,2=client writeable ranges,3=default file layouts on dirs,4=dir inode in separate object,5=mds uses versioned encoding,6=dirfrag is stored in omap,8=no anchor table,9=file layout v2,10=snapreal
m v2}
legacy client fscid: 1

Filesystem 'cephfs' (1)
fs_name cephfs
epoch   2
flags   12
created 2024-02-15T18:22:55.695260+0000
modified        2024-02-15T18:22:55.695260+0000
tableserver     0
root    0
session_timeout 60
session_autoclose       300
max_file_size   1099511627776
required_client_features        {}
last_failure    0
last_failure_osd_epoch  0
compat  compat={},rocompat={},incompat={1=base v0.20,2=client writeable ranges,3=default file layouts on dirs,4=dir inode in separate object,5=mds uses versioned encoding,6=dirfrag is stored in omap,8=no anchor table,9=file layout v2,10=snapreal
m v2}
max_mds 1
in
up      {}
failed
damaged
stopped
data_pools      [3]
metadata_pool   2
inline_data     disabled
balancer
standby_count_wanted    0



2024-02-15T18:22:55.694+0000 7f2c37c18700  4 mon.smithi067@0(leader).mds e1 prepare_command done, r=0
2024-02-15T18:22:55.694+0000 7f2c37c18700 10 mon.smithi067@0(leader).paxosservice(mdsmap 1..1) dispatch forced immediate propose
2024-02-15T18:22:55.694+0000 7f2c37c18700 10 mon.smithi067@0(leader).paxosservice(mdsmap 1..1) propose_pending
2024-02-15T18:22:55.694+0000 7f2c37c18700 10 mon.smithi067@0(leader).mds e1 encode_pending e2
2024-02-15T18:22:55.694+0000 7f2c37c18700 10 mon.smithi067@0(leader) e2 log_health updated 2 previous 0
2024-02-15T18:22:55.694+0000 7f2c37c18700 -1 log_channel(cluster) log [ERR] : Health check failed: 1 filesystem is offline (MDS_ALL_DOWN)
```

~3s of delay when the MDS was probably created., but this is expected as you mention, but what else changed to have this warning consistently show up now...

Reply:
The MDS were deployed ~9s later.

```
2024-02-15T18:23:02.315+0000 7f47dc273780  1 -- 172.21.15.174:0/3228562887 >> [v2:172.21.15.174:3300/0,v1:172.21.15.174:6789/0] conn(0x561b3a2e8c00 msgr2=0x561b3a330300 secure :-1 s=STATE_CONNECTION_ESTABLISHED l=1).mark_down
2024-02-15T18:23:02.315+0000 7f47dc273780  1 --2- 172.21.15.174:0/3228562887 >> [v2:172.21.15.174:3300/0,v1:172.21.15.174:6789/0] conn(0x561b3a2e8c00 0x561b3a330300 secure :-1 s=READY pgs=32 cs=0 l=1 rev1=1 rx=0x561b3afc0750 tx=0x561b3afca120).stop
2024-02-15T18:23:02.315+0000 7f47dc273780  1 -- 172.21.15.174:0/3228562887 shutdown_connections
2024-02-15T18:23:02.315+0000 7f47dc273780  1 --2- 172.21.15.174:0/3228562887 >> [v2:172.21.15.67:3300/0,v1:172.21.15.67:6789/0] conn(0x561b3a2e9000 0x561b3a330800 unknown :-1 s=CLOSED pgs=0 cs=0 l=0 rev1=0 rx=0 tx=0).stop
2024-02-15T18:23:02.315+0000 7f47dc273780  1 --2- 172.21.15.174:0/3228562887 >> [v2:172.21.15.174:3300/0,v1:172.21.15.174:6789/0] conn(0x561b3a2e8c00 0x561b3a330300 unknown :-1 s=CLOSED pgs=32 cs=0 l=1 rev1=1 rx=0 tx=0).stop
2024-02-15T18:23:02.315+0000 7f47dc273780  1 -- 172.21.15.174:0/3228562887 >> 172.21.15.174:0/3228562887 conn(0x561b3a2e8800 msgr2=0x561b3a3ce000 unknown :-1 s=STATE_NONE l=0).mark_down
2024-02-15T18:23:02.315+0000 7f47dc273780  1 -- 172.21.15.174:0/3228562887 shutdown_connections
2024-02-15T18:23:02.315+0000 7f47dc273780  1 -- 172.21.15.174:0/3228562887 wait complete.
2024-02-15T18:23:02.315+0000 7f47dc273780  0 set uid:gid to 167:167 (ceph:ceph)
2024-02-15T18:23:02.315+0000 7f47dc273780  0 ceph version 16.2.4 (3cbe25cde3cfa028984618ad32de9edc4c1eaed0) pacific (stable), process ceph-mds, pid 7
```

cephadm got a _bit_ slower?

Reply:
There is also

> Replacing daemon mds.cephfs.smithi038.jgrryv as rank 0

warning in the fs:upgrade test which is also hitting in pacific: https://github.com/ceph/ceph/pull/55523#issuecomment-1947304053

Reply:
Probably a fallout from https://github.com/ceph/ceph/pull/54312/